app/models/spree/user_decorator.rb

```diff
@@ -0,0 +1,4 @@
+Spree.user_class.class_eval do
+  has_one :braintree_customer, class_name: 'SolidusPaypalBraintree::Customer',
```

Comment:
This might benefit from having an `index` as well. Are we ever finding a user by this field?

Reply:
I think there may be an index already on the `user_id` column of the `SolidusPaypalBraintree::Customer`